aggregate-snapshot/aggregate-snapshot-service/src/main/java/uk/gov/justice/services/eventsourcing/source/core/SnapshotAwareEventSource.java

```diff
@@ -34,6 +39,13 @@ public EventStream getStreamById(final UUID streamId) {
         return new SnapshotAwareEnvelopeEventStream(streamId, eventStreamManager, snapshotService);
     }
 
+    @Override
+    public Stream<EventStream> getEventStreamsBySequence(final long sequenceNumber) {
+        return eventStreamJdbcRepository
+                .findAllGreaterThanAndEqualToSequenceNumber(sequenceNumber)
```

Comment:
GreaterThan Or Equal?
But maybe this can be renamed clearer also.

Reply:
Changed to getBySequenceId as we  have now changing it to use the EventStreamRepository interface instead of the implementation